content/apidocs-mxsdk/apidocs/old-version/team-server-api.md

```diff
@@ -0,0 +1,206 @@
+---
```

Comment:
Let's check if we need to keep the old version.

Reply:
I added a warning on top that informs the reader about the plan to deprecate the current API and links to the new API

Reply:
I created a new link in the index page about the APIs. Now there is a link to the `normal` API and a second one that points to the `previous` implementation. The `previous` implementations now shows a warning about the incoming deprecation and links the new APIs.

Reply:
I would add a notice on top of the new documentation that links to the old version. People having bookmarked the current documentation may otherwise wonder where it went.

Reply:
Added the new notice in the new documentation that links to the old version.